rest_lights.cpp

```diff
@@ -2381,10 +2381,10 @@ int DeRestPluginPrivate::setLightAttributes(const ApiRequest &req, ApiResponse &
         //taskRef.transitionTime = 4;
         //taskRef.onTime = 0;
 
-        QByteArray direction = QByteArray("\x01\x00", 2);
+        QByteArray direction = QByteArray("\x00\x00", 2);

```

Comment:
I really would like DS82 support to be merged, so do not consider it as a critique.

commit subject and commit  message should really say what patch is doing and why it's changing theses lines.
That will help maintainer to understand its purpose and why it's needed.

Reply:
Is written on the message ?

>Correct the "reverse" command for all tuya covering using the tuya cluster.

Reply:
yes, something along these lines
subject could be short like you suggest

commit message could be multiline a provide more verbose description what commit does an why it does something. 
the same applies to other commits.

it helps not only maintainer but also documents why/what change does, so others (including yourself in the future when you forget about it) can understand purpose of the commit.